packages/compiler/test/checker/check-parse-errors.ts

```diff
@@ -19,7 +19,7 @@ describe("compiler: semantic checks on source with parse errors", () => {
 
     const diagnostics = await testHost.diagnose("./");
     strictEqual(diagnostics.length, 5);
-    match(diagnostics[0].message, /Property expected/);
+    match(diagnostics[0].message, /'}' expected/);
```

Comment:
do you also get as `')' expected` now if you are not closing an operation 

```
op foo(abc: string, some: string
```

Reply:
Seems I got something wrong here, it works for empty list only.

![image](https://user-images.githubusercontent.com/75470/154120768-0885c0d6-7220-4ece-addf-3ced55e15fb1.png)

Reply:
Great question!

Reply:
interesting